userspace/libscap/scap.c

```diff
@@ -37,6 +39,20 @@ along with sysdig.  If not, see <http://www.gnu.org/licenses/>.
 #include "scap_savefile.h"
 #include "scap-int.h"
 
+#ifdef HAS_ANALYZER
+
```

Comment:
Here we may use: `#if defined(HAS_ANALYZER) && defined(HAS_THREAD_FILTERING)` to reduce nested IFDEFs